server/api/user/userController.js

```diff
@@ -63,6 +69,17 @@ exports.update = function(req, res, next) {
 };
 
 exports.destroy = function(req, res, next) {
-
-  res.send('DELETE user with ID ' + req.params.id);
+  var username = req.params.username;
+  User.findOne({username: username}, function(err, user) {
+    if(!user) {
+      res.status(404).end();
+    } else {
+      User.remove({username: username}, function(err) {
+        if(err) {
+          res.status(500).end();
+        }
+        res.status(200).end();
```

Comment:
Is 200 definitely the right code? Not sure, but maybe there's something more specialized?